index.js

```diff
@@ -117,6 +117,13 @@ class LogstashUDP extends Transport {
         ? options.connFlushInterval
         : 10000;
 
+    const defaultFormat = winston.format.printf(this._buildLog.bind(this));
+    if (!this.format) {
+      this.format = defaultFormat;
+    } else {
+      this.format = winston.format.combine(this.format, defaultFormat);
```

Comment:
Just in case someone passes a custom format in the options, merge it with the udp transport format.